static/app/views/performance/traceDetails/utils.tsx

```diff
@@ -31,6 +33,7 @@ export function getTraceDetailsUrl({
   source,
   view,
 }: {
+  // @TODO add a type for dateSelection
```

Comment:
Maybe try `PageFilters['datetime']`

Reply:
I tried and there are a few places where it breaks because of statsPeriod being set. cc @Abdkhan14, maybe you know what type would be appropriate here